loader.ts

```diff
@@ -96,7 +97,7 @@ export class Loader {
       case "file:":
       case "https:":
       case "http:": {
-        const wasmResponse = await fetch(url);
+        const wasmResponse = await fetchWithRetries(url);
```

Comment:
This was previously missing a `wasmResponse.ok` check leading to a cryptic error message.